libs/island-ui/core/src/lib/Input/types.ts

```diff
@@ -61,6 +61,7 @@ export interface InputProps extends InputComponentProps {
   errorMessage?: string
   tooltip?: string
   backgroundColor?: ResponsiveProp<InputBackgroundColor>
+  prefix?: string
```

Comment:
I wonder if this feature is generally useful enough to extend our Input component. It's also arguably confusing for screen readers.

An alternative implementation would be to automatically prefill the id based on the name INCLUDING the prefix. If the user changes the id manually and tries to remove the prefix, they get a validation error. It would not match the two colour design, but maybe that's fine for simplicity's sake?

Reply:
I guess my main concern is the lack of oversight and direction in the design system / island-ui project. I'd want some collaboration and discussion to take place before adding features like this.

Otherwise our components may bloat up if everyone adds any new design idea to it.

Reply:
I started out that way. But I thought it would bring to much complexity for a thing that can be solved much simpler. That way you need to make sure that the prefix is always present and cannot be erased in my mind

Reply:
I agree with @eirikurn, I think there needs to be a discussion with the designers before implementing. things like this. Otherwise the components in Island-UI will become bloated and hard to maintain. Also, the designers should have a say in what we implement IMO.

You could implement this by using a React-Input-Mask (already included in the monorepo). You wouldn't get the difference in colour though.

https://user-images.githubusercontent.com/3789875/223674175-129b4c9f-260e-476e-817f-0fc5ecd73252.mov 

The code is 

```
<InputMask
        mask="PREFIX 999-9999"
        maskPlaceholder={null}
        alwaysShowMask={true}
        ...
```

Reply:
Agreed. Thats a great suggestion. Thanks

Reply:
Solved another way out side component since mask was not sufficient for this scenario